commcare_connect/templates/tailwind/components/sidenav-items.html

```diff
@@ -0,0 +1,25 @@
+<a href="{{ target }}" class="flex rounded-lg items-center p-2 hover:bg-brand-sunset-light transition-all duration-300 relative group">
+    <!-- Icon Container with Fixed Width -->
+    <div class="flex items-center justify-center w-8 h-8">
+        <i class="fa-light fa-{{ icon }}"></i>
+    </div>
+
+    <!-- Text Container with Fixed Width -->
+    <div class="transition-all duration-300 overflow-hidden flex items-center"
+         :class="sidebarOpen ? 'w-40 ml-4 opacity-100' : 'w-0 opacity-0'">
+        <span class="text-xs whitespace-nowrap">{{ name }}</span>
+    </div>
+
+    <!-- Hover Tooltip -->
+    <div class="absolute hidden px-2 py-1 ml-4 text-sm transform -translate-y-1/2 bg-white rounded top-1/2 left-full text-brand-deep-purple group-hover:block whitespace-nowrap z-30 shadow-lg"
+         x-show="!sidebarOpen"
+         x-cloak>
+        {{ name }}
+    </div>
+
+    <!-- Selected -->
+     <!-- {% if sidenav_active == name %} -->
```

Comment:
Should this be commented out?

Reply:
I commented it out when I tried to test active state, forgot to uncomment. Done that